system/lib/libc/emscripten_get_heap_size.c

```diff
@@ -5,8 +5,8 @@
  * found in the LICENSE file.
  */
 
-#include <emscripten/heap.h>
+#include <stddef.h>
 
 size_t emscripten_get_heap_size() {
-  return __builtin_wasm_memory_size(0) * WASM_PAGE_SIZE;
+  return __builtin_wasm_memory_size(0) << 16;
```

Comment:
Won't to optimizer take care of doing this?

Reply:
I don't know, but it certainly won't change a << 16 back to a * 65536. I don't think it is that good to rely on optimizer to achieve what one wants, but better to just write out what one wants directly when it is trivially possible.